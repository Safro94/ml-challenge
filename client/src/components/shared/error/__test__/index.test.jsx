import React from 'react';

import { render } from '@testing-library/react';

import Error from '../';

describe('Error', () => {
	it('should render OK', () => {
		//Act
		const { container } = render(<Error />);

		//Assert
		expect(container).toMatchSnapshot();
	});
});
