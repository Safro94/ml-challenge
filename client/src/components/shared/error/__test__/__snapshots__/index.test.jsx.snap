// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render OK 1`] = `
<div>
  <div
    class="container"
    role="alert"
  >
    <img
      alt="Error"
      class="image"
      src="/assets/error.png"
    />
    <h2>
      Ooops, ha habido un error, por favor volv√© a intentarlo mas tarde
    </h2>
  </div>
</div>
`;
